docs-ref-services/latest/hdinsight.md

```diff
@@ -381,7 +381,7 @@ client.extensions().disableMonitoring("<Resource Group Name>", "<Cluster Name>")
 
 HDInsight provides a configuration method called script actions that invokes custom scripts to customize the cluster.
 > [!NOTE]
-> More information on how to use script actions can be found here: [Customize Linux-based HDInsight clusters using script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux)
+> More information on how to use script actions can be found here: [Customize Linux-based HDInsight clusters using script actions](/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux)
```

Comment:
I think that this changed link does need to include the /en-us/ at the front: "/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux"